site/app/models/gradeable/Gradeable.php

```diff
@@ -84,141 +84,238 @@
  * @method object[] getPeerGradingPairs()
  */
 class Gradeable extends AbstractModel {
+
     /* Enum range for grader_assignment_method */
     const ROTATING_SECTION = 0;
     const REGISTRATION_SECTION = 1;
     const ALL_ACCESS = 2;
 
     /* Properties for all types of gradeables */
 
-    /** @prop @var string The course-wide unique gradeable id */
```

Comment:
Please revert these spacing changes (I would recommend grabbing your non-spacing changes, reverting the Gradeable.php file, and copying the changes back in).